utils.py

```diff
@@ -37,6 +37,60 @@ def getScript(scriptname):
 
 # BUILD HELPERS
 
+# Helper function that creates a worker instance.
+def createWorker(worker_name_prefix, worker_id, worker_type, dockerfile, jobs=5, save_packages=False, shm_size='15G', worker_name_suffix=''):
+    worker_name = worker_name_prefix + str(worker_id) + '-docker'
+    name = worker_name + worker_type
+
+    i = worker_id
+    tls = None
+
+    if worker_name_prefix.startswith('hz'):
+        b_name = 'x64-bbw'
+    elif worker_name_prefix.startswith('intel'):
+        b_name = 'x64-bbw'
+    elif worker_name_prefix.startswith('p9'):
+        b_name = 'p9-bbw'
+    elif worker_name_prefix.startswith('amd'):
+        b_name = 'x64-bbw'
+    else:
+        b_name = worker_name_prefix
+    base_name = b_name + '-docker' + worker_type
+
+    volumes=['/srv/buildbot/ccache:/mnt/ccache', '/srv/buildbot/packages:/mnt/packages', '/mnt/autofs/master_packages/:/packages']
+    # Set master FQDN - for VPN machines it should be 100.64.100.1
```

Comment:
I think that we could manage this by editing the `/etc/hosts` file with ansible.

Reply:
Yep, probably a good idea

Reply:
I suggest you keep it that way for the moment because I want to take time to implement this correctly, then I will restrict public access to only listed machines that don't connect through VPN. This will also make sure that the /etc/hosts resolution is working as expected.